readme.md

```diff
@@ -17,8 +17,13 @@ var splitLines = require('split-lines');
 
 splitLines('foo\r\nbar\r\nbaz\nrainbow');
 //=> ['foo', 'bar', 'baz', 'rainbow']
+
+splitLines('foo\r\nbar\r\nbaz\nrainbow', true);
```

Comment:
@jamestalmage It's the same case as using URL incorrectly instead of URI. Nobody knows the difference and it doesn't matter. Just be clear in the examples what a newline is.